app/Http/Controllers/PaymentController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Enums\PaymentStatus;
+use App\Models\Order;
+use Illuminate\Http\Request;
+use Inertia\Inertia;
+
+class PaymentController extends Controller
+{
+    public function success(Request $request, Order $order = null)
+    {
+        $order->update([
+            'payment_status' => PaymentStatus::SUCCESS->value
+        ]);
+
+        return Inertia::render('Payment/success');
+    }
+
+    public function failed()
+    {
+        return Inertia::render('Payment/failed');
+    }
```

Comment:
Like with the success handler, it's best if we also update the order when it's failed so that we may keep track of it.